projects/docs/docs/guides/environment.md

```diff
@@ -18,6 +18,13 @@ If you want to pass multiple environment variables just separate them with a spa
 TUIST_APP_NAME=MyApp TUIST_APP_LOCALE=pl tuist generate
 ```
 
+Tuist also supports loading in environment variables from a `.env` file located in the `Tuist` folder. Simply create a `.env` file in `Tuist` and prefix your environment variables using the standard Tuist convention i.e. `TUIST_XXX`:
+
+```bash
+TUIST_APP_NAME=MyApp
+TUIST_APP_LOCALE=pl
+```
```

Comment:
Given we are explicitly loading this file, we can remove the `TUIST_` from here, and just add it when loading the variables from file

Reply:
I don't know if that makes sense and also isn't in line with the rest of Tuist. If the dotenv implementation loads the variables directly into `ProcessInfo.processInfo`, how do we then disambiguate between those from the process and those added via the `.env`? It would require more state when this solution should ideally be stateless.

Reply:
I'd probably be explicit here as @thebarndog. We can also allow for both and add `TUIST` prefix only when necessary but would rather keep a single solution 😁

Reply:
it's a bit weird that you are forced to add TUIST in front of all of them.
However, if we think the .env file could possibliy contain variables not related to tuist, it would then make sense so you can separate the tuist ones from the others 👍